lib/kino/process.ex

```diff
@@ -421,15 +423,36 @@ defmodule Kino.Process do
     {func_result, sequence_diagram}
   end
 
-  defp generate_participant_entry(pid, idx) do
+  # :proc_lib.get_label/1 was added in OTP 27
+  if Code.ensure_loaded?(:proc_lib) and function_exported?(:proc_lib, :get_label, 1) do
+    defp get_label(pid), do: :proc_lib.get_label(pid)
+  else
+    defp get_label(_pid), do: :undefined
+  end
+
+  defp generate_participant_entry(pid, idx, process_label) do
     try do
       {:registered_name, name} = process_info(pid, :registered_name)
       "participant #{idx} AS #{module_or_atom_to_string(name)};"
     rescue
-      _ -> "participant #{idx} AS #35;PID#{:erlang.pid_to_list(pid)};"
+      _ ->
+        case process_label do
+          :undefined ->
+            "participant #{idx} AS #35;PID#{:erlang.pid_to_list(pid)};"
+
+          process_label ->
+            "participant #{idx} AS #{format_for_mermaid_participant_alias(pid, process_label)};"
+        end
     end
   end
 
+  defp format_for_mermaid_participant_alias(pid, process_label) do
+    pid_text = :erlang.pid_to_list(pid) |> List.to_string()
+
+    participant_alias = "#{inspect(process_label)}<br/>#{pid_text}"
+    String.replace(participant_alias, "\"", "")
```

Comment:
Should this also be `format_as_mermaid_unicode_text`? If so, let's move the replace into that function (my understanding is that we remove `"` because we want to enclose the text in `"`, so it's related).

Reply:
That `String.replace` is to remove the quotes that comes when we `inspect(...)` a string.

Otherwise, if the process label has a string set as label, the label in the diagram would include the quotes, like that:

![CleanShot 2024-06-13 at 10 45 07](https://github.com/livebook-dev/kino/assets/2719/9b692592-f9d4-4423-acad-9bd7963b0228)

The other `format_as_mermaid_unicode_text` is there because if one uses unicode chars inside the process label, mermaid needs to enclose the text with `"` to be able to display unicode text: https://mermaid.js.org/syntax/flowchart.html#unicode-text